homeassistant/components/nut/const.py

```diff
@@ -38,3 +40,67 @@
     "FSD": "Forced Shutdown",
     "ALARM": "Alarm",
 }
+
+COMMAND_BEEPER_DISABLE = "beeper.disable"
+COMMAND_BEEPER_ENABLE = "beeper.enable"
+COMMAND_BEEPER_MUTE = "beeper.mute"
+COMMAND_BEEPER_TOGGLE = "beeper.toggle"
+COMMAND_BYPASS_START = "bypass.start"
+COMMAND_BYPASS_STOP = "bypass.stop"
+COMMAND_CALIBRATE_START = "calibrate.start"
+COMMAND_CALIBRATE_STOP = "calibrate.stop"
+COMMAND_LOAD_OFF = "load.off"
+COMMAND_LOAD_ON = "load.on"
+COMMAND_OUTLET_N_LOAD_CYCLE = "outlet.n.load.cycle"
+COMMAND_OUTLET_N_LOAD_OFF = "outlet.n.load.off"
+COMMAND_OUTLET_N_LOAD_ON = "outlet.n.load.on"
+COMMAND_OUTLET_N_SHUTDOWN_RETURN = "outlet.n.shutdown.return"
```

Comment:
I think these take a specific outlet as the N value?

Reply:
You are right, my bad. These were not supported by my unit and I missed the parameter from the docs.